libs/plutus-preprocessor/src/Main.hs

```diff
@@ -152,9 +152,10 @@ main = do
       "import Cardano.Ledger.Alonzo.Language (Language (..))",
```

Comment:
I don't know how this `Main` gets called, or how to test it..  I had to "compile in my head"  and I have no confidence it will work. Any idea how to check ?

Reply:
See here for explanation of how to do it (last line) https://github.com/input-output-hk/cardano-ledger/pull/2328